zebrad/tests/acceptance.rs

```diff
@@ -2606,3 +2605,86 @@ async fn state_format_test(
     }
     Ok(())
 }
+
+/// Snapshot the `z_getsubtreesbyindex` method in a synchronized chain.
+#[tokio::test]
+#[ignore]
+async fn rpc_z_getsubtreesbyindex_sync_snapshots() -> Result<()> {
```

Comment:
This name change is part of running both tests at the same time.

```suggestion
/// Snapshot the `z_getsubtreesbyindex` method in a synchronized chain.
///
/// This test name must have the same prefix as the `fully_synced_rpc_test`, so they can be run in the same test job.
#[tokio::test]
#[ignore]
async fn fully_synced_rpc_z_getsubtreesbyindex_snapshot_test() -> Result<()> {
```

Reply:
Done at https://github.com/ZcashFoundation/zebra/pull/7515/commits/2ebb28032288e41ea80176b1ddcee59208cfcb28